### PR TITLE
fix(content-group): card copy fix

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)

#4393

### Description

Fixed the `copy` variable for Card component in the Content Group Simple story in which it wasn't being accessed correctly, and added the `ArrowRight20` icons to the footer as in the React version.

### Changelog

**New**

- added back the `ArrowRight20` icon to the Card footer

**Changed**

- mislabeled `TextCTA` argument to `CardCTA`

### CODE REVIEW

**Review by ariellalgilmore — COMMENTED** [review]
LGTM thanks @IgnacioBecerra! Can we also update the documention and example?

File: packages/web-components/src/components/content-group-simple/__stories__/content-group-simple.stories.ts [hunk 1]
```diff
@@ -155,15 +158,18 @@ export const WithImage = ({ parameters }) => {
...
       </dds-card-cta>
     </dds-content-group-simple>
   `;
 };
 
 export const WithVideo = ({ parameters }) => {
   const { copy: groupCopy, heading: groupHeading } = parameters?.props?.ContentGroupSimple ?? {};
-  const { copy: ctaCopy, ctaType, href } = parameters?.props?.TextCTA ?? {};
+  const { copy: ctaCopy, ctaType, href } = parameters?.props?.CardCTA ?? {};
+  console.log(ctaCopy);
```
Comment (asudoh) [inline packages/web-components/src/components/content-group-simple/__stories__/content-group-simple.stories.ts:172]:
```suggestion
```

**Review by asudoh — COMMENTED** [review]
Thank you for jumping in @IgnacioBecerra!

File: packages/web-components/src/components/content-group-simple/__stories__/content-group-simple.stories.ts (hunk 1 above)
Comment (IgnacioBecerra) [reply packages/web-components/src/components/content-group-simple/__stories__/content-group-simple.stories.ts:172]: Thanks for the catch @asudoh!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @IgnacioBecerra!

**Review by kennylam — APPROVED** [review]
LGTM thanks @IgnacioBecerra!

**Pull request merged**